packages/wallets/bitkeep/package.json

```diff
@@ -38,6 +38,7 @@
         "@solana/wallet-adapter-base": "workspace:^"
     },
     "devDependencies": {
+        "@bitget-wallet/web3-sdk": "^0.0.8",
```

Comment:
Please move to dependencies, since this is used by the adapter code (not just build/test tooling).

Reply:
Understood. I will set up a new PR to address this, as the colleague who originally filed the PR is no longer with the company.